frontend/assessments/1/style.css

```diff
@@ -0,0 +1,215 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font-family: "Work sans";
+}
+
+#main #top-header {
+  background-color: rgb(255, 179, 80);
```

Comment:
Store the colors/hard-coded values as the constants in different file and import it